README.md

```diff
@@ -90,3 +95,27 @@ You can also:
 # FAQ
 
 Check out https://wasp-lang.dev/#faq.
+
+# License
```

Comment:
We already have License as a separate file (LICENSE), so we can remove this right?